flask_api.py

```diff
@@ -69,7 +69,7 @@ def handle_question():
     return jsonify(response), SUCCESS
 
 
-@app.route('/new_data/wakeword', methods=['POST'])
+@app.route('/new_data/audiosample', methods=['POST'])
```

Comment:
let's go with `/audiosample` to keep it simple

Reply:
# @Waidhoferj what are your thoughts?

Reply:
I have no preference either way.